core/src/banking_stage/latest_unprocessed_votes.rs

```diff
@@ -256,7 +256,7 @@ impl LatestUnprocessedVotes {
                 if Self::allow_update(&vote, latest_vote, should_replenish_taken_votes) {
                     let old_vote = std::mem::replace(latest_vote, vote);
                     if old_vote.is_vote_taken() {
-                        self.num_unprocessed_votes.fetch_add(1, Ordering::Relaxed);
+                        self.num_unprocessed_votes += 1;
```

Comment:
nit: saturating_add?

Reply:
I just took it as it was wrapping math before, without issue, so it can be wrapping math now?

Reply:
good point, on second thought I actually don't know if saturating is better than wrapping here. Keeping same behavior is good for this PR.